File: 백준/Bronze/24883. 자동완성/자동완성.cs
using System;

class Q24883
{
    static void Main(string[] args)
    {
        char ch = Char.Parse(Console.ReadLine());
        
        if (ch == 'N' || ch == 'n') {
            Console.WriteLine("Naver D2");
        } else {
            Console.WriteLine("Naver Whale");
        }
    }
}

File: 백준/Bronze/24883. 자동완성/자동완성.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        char ch = sc.next().charAt(0);
        
        if (ch == 'N' || ch == 'n') {
            System.out.println("Naver D2");
        } else {
            System.out.println("Naver Whale");
        }
    }
}